Packs/CheckPoint_FW/Integrations/CheckPoint_FW/README.md

```diff
@@ -38,13 +75,13 @@ Show all host objects
 
 | **Path** | **Type** | **Description** |
 | --- | --- | --- |
-| checkpoint.host.name | String | object name | 
-| checkpoint.host.uid | String | object uid | 
-| checkpoint.host.type | String | object type | 
+| CheckPoint.host.name | String | object name | 
+| CheckPoint.host.uid | String | object uid | 
+| CheckPoint.host.type | String | object type | 
 
 
 #### Command Example
-``` ```
+!checkpoint-host-list
 
 #### Human Readable Output
```

Comment:
have you regenerated the file as I asked? where are the outputs of the commands (human-readable and context)?